glmtools/glm.py

```diff
@@ -53,5 +61,33 @@ def t_test(c, X, B, sigma_2, df):
     p : array shape (V,)
         two-tailed probability value for each t statistic.
     """
-    # Your code code here
-    return
+
+    #get t statistic
+    c_b_cov = c.dot(npl.pinv(X.T.dot(X))).dot(c)
+    t = c.dot(B) / np.sqrt(sigma_2 * c_b_cov)
+
+    #get two-tailed p-value for t statistic
+    t_dist = scipy.stats.t(df=df)
+    if t > 0:
```

Comment:
This will only work for a single p value, and therefore a single column of Y.  Try adding a test for multiple columns of y.